eval/python/distance.py

```diff
@@ -1,19 +1,26 @@
 import argparse
 import numpy as np
-import sys
+# import sys
+np.seterr(divide='ignore', invalid='ignore')  # fix runtime error when dividing by zero
+
 
 def generate():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--vocab_file', default='vocab.txt', type=str)
+    parser.add_argument('--vocab_file', default=None, type=str)
     parser.add_argument('--vectors_file', default='vectors.txt', type=str)
     args = parser.parse_args()
 
-    with open(args.vocab_file, 'r') as f:
-        words = [x.rstrip().split(' ')[0] for x in f.readlines()]
+    if args.vocab_file is not None:
+        with open(args.vocab_file, 'r') as f:
+            words = [x.rstrip().split(' ')[0] for x in f.readlines()]
     with open(args.vectors_file, 'r') as f:
         vectors = {}
+        words = []
```

Comment:
It looks like this line is clobbering `words` if it was successfully read at line 15